### PR TITLE
add `--ignore_missing_core_models` CLI flag to bypass checking for missing core models

### PR DESCRIPTION
**Pull request by damian0815**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Community Node Submission

As per title - if InvokeAI won't start because you don't have some core conversion models installed, **and you know you will not need them**, you can pass `--ignore_missing_core_models` on startup and the check will be bypassed.

### CODE REVIEW

File: invokeai/backend/install/check_root.py [hunk 1]
```diff
@@ -32,5 +33,6 @@ def check_invokeai_root(config: InvokeAIAppConfig):
         print(
             '** From the command line, activate the virtual environment and run "invokeai-configure --yes --skip-sd-weights" **'
         )
+        print('** (To skip this check completely, add "--ignore_missing_core_models" to your CLI args. Not recommended.)')
```
Comment (lstein) [inline invokeai/backend/install/check_root.py:36]: Could you change the user message to tell them the consequences of skipping this check? 

"Not installing these core models will prevent the loading of some or all .safetensors and .ckpt files. You can always come back and install these core models in the future."

**Review by lstein — CHANGES_REQUESTED** [review]
Looks good. Just need to tell the user the consequence of not installing the core models as noted in the code comment above.

File: invokeai/backend/install/check_root.py (hunk 1 above)
Comment (damian0815) [reply invokeai/backend/install/check_root.py]: done

**Review by psychedelicious — APPROVED** [review]

**Comment by psychedelicious** [comment]
@lstein Need you to re-approve now that your requested changes have been addressed.

**Review by lstein — APPROVED** [review]
LGTM

**Pull request merged**